vars/goMake.groovy

```diff
@@ -10,6 +10,8 @@ def call(body) {
     def ghOrg =  config.githubOrganisation
     def dockerOrg = config.dockerOrganisation
     def prj = config.project
+    def makeCommand = config.makeCommand ?: 'make'
+    def installPreRequisites = config.installPreRequisites
 
```

Comment:
https://github.com/rupalibehera/kompose/blob/pr-pipeline/Jenkinsfile#L9 , The test needs jq (https://stedolan.github.io/jq/) to be installed or else they fail.